src/Skeletor/App/Config/skeletor.yml

```diff
@@ -1,11 +1,11 @@
 config:
-    name: Skeletor CLI
+    name: 'Skeletor CLI'
     version: 1.0.0
```

Comment:
I wonder if there is a nice way to get the version number from the git tag automatically?